src/olympia/abuse/cinder.py

```diff
@@ -197,17 +213,29 @@ def __init__(self, addon, version=None):
 
     @property
     def id(self):
-        return str(self.addon.id)
+        return self.get_str(self.addon.id)
 
     def get_attributes(self):
+        # FIXME: translate translated fields in reporter's locale, send as
+        # dictionaries?
+        version = self.addon.current_version
         return {
             'id': self.id,
+            'average_daily_users': self.addon.average_daily_users,
+            'description': self.get_str(self.addon.description),
             'guid': self.addon.guid,
+            'homepage': self.get_str(self.addon.homepage),
+            'last_updated': self.get_str(self.addon.last_updated),
+            'name': self.get_str(self.addon.name),
+            'release_notes': self.get_str(version.release_notes) if version else '',
             'slug': self.addon.slug,
-            'name': str(self.addon.name),
+            'summary': self.get_str(self.addon.summary),
+            'support_email': self.get_str(self.addon.support_email),
+            'support_url': self.get_str(self.addon.support_url),
+            'version': self.get_str(version.version) if version else '',
```

Comment:
The use-case I was seeing is that if you are reporting a user because you are on their user profile page... then you're seeing their add-ons, and might be basing your report on the names/summaries being displayed... I guess I could move `description`, `support_email`, `support_url`, `release_notes` for sure though.